config/helm/deployment-args.yaml

```diff
@@ -14,6 +14,7 @@ spec:
             - --feature-gates=MachinePool=true,ClusterResourceSet=true,ClusterTopology=true
             - --watch-filter={{ .Values.watchFilter }}
             - --v=0
+          image: '{{.Values.images.domain}}/{{.Values.images.core.name}}:{{.Values.images.core.tag | default .Values.images.tag}}'
```

Comment:
We have the images defined in the `kustomization.yaml` file. I think now it's confusing as we have the images defined here and there.